scripts/app.js

```diff
@@ -0,0 +1,31 @@
+var novels = [];
+
+function Article (books) {

```

Comment:
I suggest using loop to iterate through all the book object properties because down the line if you have hundreds of object properties, adding a line of `this.property = books.property` will make your constructor function really large.
Here is something I did to my constructor function
```
function Project (projectData) {
  for (var property in projectData) {
    this[property] = projectData[property];
  }
}
```